packages/react-core/src/components/NotificationBadge/__tests__/NotificationBadge.test.tsx

```diff
@@ -40,3 +40,17 @@ test('Renders with .pf-m-clicked when isExpanded is passed in.', () => {
 
   expect(screen.getByRole('button')).toHaveClass('pf-m-clicked');
 });
+
+test('Renders with .pf-m-notify when hasAnimation is passed in.', () => {
+  render(<NotificationBadge hasAnimation />);
+
+  expect(screen.getByRole('button')).toHaveClass('pf-m-notify');
+});
```

Comment:
Can we import the notif badge style object and use that instead of hardcoding the class names

Also can we just add one more test to check that the notify class is not applied by default/when the new prop isn't passed

Reply:
i was following the model of the other tests and didn't want to mix and match... so i imported the style object but updated the legacy tests that were hardcoding the classnames.